### PR TITLE
S0_box_88_veilhelmAlex

### PR DESCRIPTION
**Pull request by veilhelm**
added some UX functionality to the login. (error message when Axios call fails, automated cleaning of fields on error and autofocus on Email field) 
Routing functionality added to the app

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,15 +1,24 @@
...
 import './App.css';
 import Login from "./pages/Login"
-import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom"
+import { BrowserRouter as Router, Switch, Route, Redirect } from "react-router-dom"
+
 
 
 
 function App() {
+  //const login = <Login></Login>
```
Comment (simonhoyos) [inline src/App.js]: This is dead code. If it is not in use, then you can remove it.

File: src/App.js [hunk 2]
```diff
@@ -1,15 +1,24 @@
...
   return (
     <Router>
     <div className="App">
-      <Login></Login>
+      <Switch>
+        <Route exact path="/lender/login" component={Login} />
+        {/* <Route exact path="/lender/register/" component={register} /> */}
```
Comment (simonhoyos) [inline src/App.js:15]: Same as before

File: src/components/login/login_form.js [hunk 3]
```diff
@@ -1,34 +1,59 @@
...
+                     email,
+                     password
+                 }
+             })
+             console.log(token)
+        }catch(error) {
+            props.handleErrorLogin(error.response.data)
+            clearFields([base.emailId, base.passwordId])
+            clearState()
+            document.getElementById(base.emailId).focus()
```
Comment (simonhoyos) [inline src/components/login/login_form.js:46]: For now, it's ok to have it like this. I recommend you to read this section of the React documentation to handle this type of behavior [Refs and the DOM](https://reactjs.org/docs/refs-and-the-dom.html)

File: src/pages/Login.js [hunk 4]
```diff
@@ -3,17 +3,25 @@ import LoginForm from "../components/login/login_form"
 
 class Login extends React.Component{
     state = {
-        loginError : false
+        loginError : false,
+        errMessage: ""
     }
     
+    //change the state of error in login
```
Comment (simonhoyos) [inline src/pages/Login.js]: Comments usually add complexity to code. In the future, if you have to make a change in the code, you'll have to maintain the code and the comments. Most of the times code can be self explanatory.

**Review by simonhoyos — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (veilhelm) [reply src/App.js]: ok, will delete then.

File: src/App.js (hunk 2 above)
Comment (veilhelm) [reply src/App.js:15]: got it!

File: src/components/login/login_form.js (hunk 3 above)
Comment (veilhelm) [reply src/components/login/login_form.js:46]: ok, I will reed the documentation and implement the changes in the next commit.

File: src/pages/Login.js (hunk 4 above)
Comment (veilhelm) [reply src/pages/Login.js]: ok got it! but, if I am implementing something that my team is not aware of or don´t know about a new syntax or functionality, how should I handle that?

**Pull request merged**